equipment_handler/src/adcs_handler.c

```diff
@@ -281,11 +281,14 @@ ADCS_returnState ADCS_load_file_download_block(uint8_t file_type, uint8_t counte
     command[0] = LOAD_FILE_DOWNLOAD_BLOCK_ID;
     command[1] = file_type;
     command[2] = counter;
-    memcpy(&command[3], &offset, 4);
-    command[7] = block_length & 0xFF;
-    command[8] = block_length >> 8;
-
-    return adcs_telecommand(command, 5); //* tested + (command[6] * 256 * 256 * 256 + command[5] *
+    command[3] = offset & 0x000000FF;
+    command[4] = (offset >> 8) & 0x000000FF;
+    command[5] = (offset >> 16) & 0x000000FF;
+    command[6] = (offset >> 24) & 0x000000FF;
+    command[7] = block_length & 0x00FF;
+    command[8] = (block_length >> 8) & 0x00FF;
+
+    return adcs_telecommand(command, 9); //* tested + (command[6] * 256 * 256 * 256 + command[5] *
```

Comment:
Please delete these "//* " comments. They were for unit test purposes.

Reply:
Will do